src/pixi/core/math/ObservablePoint.hx

```diff
@@ -0,0 +1,35 @@
+package pixi.core.math;
+
+@:native("PIXI.ObservablePoint")
+extern class ObservablePoint extends Point {
+
+	/**
+	 * The Point object represents a location in a two-dimensional coordinate system, where x represents
+	 * the horizontal axis and y represents the vertical axis.
+	 * An observable point is a point that triggers a callback when the point's position is changed.
+	 *
+	 * @class
+	 * @memberof PIXI
+	 * @param cb {Function} callback when changed
+	 * @param scope {Object} owner of callback
+	 * @param [x=0] {number} position of the point on the x axis
+	 * @param [y=0] {number} position of the point on the y axis
+	 */
+	function new(cb:Dynamic, scope:Dynamic, x:Float, y:Float);
```

Comment:
A few tips :)

The parameters `x` and `y` are optional here so you need to make them optional with `?`

Can you also try and avoid using `Dynamic` whereever possible. For example here the callback `cb` type can be `Void -> Void`.

You can check the status of the pull request after creating it where you can see the build state and checkstyle issues via coeclimate.